src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/CommonLogger.cs

```diff
@@ -11,12 +13,15 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
     /// </summary>
     public class CommonLogger : Common.ILogger
     {
-        public CommonLogger(ILogger logger)
+        private const int DefaultLogEventId = 23847;
```

Comment:
Just picked a number that is sufficiently different from all other used event IDs